packages/devtools_app/lib/src/trees.dart

```diff
@@ -43,6 +43,8 @@ abstract class TreeNode<T extends TreeNode<T>> {
 
   bool get isRoot => parent == null;
 
+  bool get isSelected => false;
```

Comment:
Should this getter be on the subclass instead of in the TreeNode base class? seems odd to have a getter return a set bool value when there aren't also methods in this TreeNode class to modify the state of `isSelected`

Reply:
I think this needs to be here since `_TreeViewItemState` uses this to determine whether or not to highlight a given `TreeNode` item.

Reply:
if that is the case, then we should move the logic for managing the state of isSelected into this class as well. Subclasses can always override and call super if they need to do additional work on changing selection, in which case we'd want to annotate with @mustCallSuper